ui/logviewer/App.jsx

```diff
@@ -252,23 +265,44 @@ class App extends React.PureComponent {
         />
         {job && (
           <div className="d-flex flex-column flex-fill">
-            <div className="run-data d-flex flex-row border-bottom border-secondary mx-1 mb-2">
-              <div className="d-flex flex-column job-data-panel">
-                <JobInfo
-                  job={job}
-                  extraFields={extraFields}
-                  revision={revision}
-                  className="list-unstyled"
-                  showJobFilters={false}
-                  currentRepo={currentRepo}
+            <Collapse isOpen={!collapseDetails}>
+              <div className="run-data d-flex flex-row border-bottom border-secondary mx-1 mb-2">
+                <div className="d-flex flex-column job-data-panel">
+                  <JobInfo
+                    job={job}
+                    extraFields={extraFields}
+                    revision={revision}
+                    className="list-unstyled"
+                    showJobFilters={false}
+                    currentRepo={currentRepo}
+                  />
+                  <JobDetails jobDetails={jobDetails} />
+                </div>
+                <ErrorLines
+                  errors={errors}
+                  onClickLine={this.setSelectedLine}
                 />
-                <JobDetails jobDetails={jobDetails} />
               </div>
-              <ErrorLines errors={errors} onClickLine={this.setSelectedLine} />
-            </div>
+            </Collapse>
+            <Button
```

Comment:
Rather than having an entire bar at the top of the page for expanding the details, let's just have a button on the top black navbar that says "Show details: ##" where ## is the number of failure lines we have available.  This would take up less screen real-estate.

Reply:
I was thinking that if we have keep those top panels shown by default maybe it'd be more intuitive to have an expand/collapse button as shown, but as a normal size button on the left side, at the bottom of the panel. I suppose this is the kind of thing where we could get user feedback :)

Reply:
Sure, expanded by default is probably fine.  I don't have too strong of an opinion on that aspect.  Agreed about user feedback.  I still feel like a button on the navbar would be better because it wouldn't eat up an additional line on the page.  But let's ask the next folks we talk to.  :)